Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -76,7 +88,7 @@ private static bool CheckActiveSession()
         /// <summary>
         /// Utility function to check if the sdk has been initiazed
         /// </summary>
-        /// <returns></returns>
+        /// <returns>True if initialized, false otherwise.</returns>
```

Comment:
More of a praxis question; should we document parameters all the time? If so, it should be added here.

Reply:
Yes, we should. I'll add it 👍